layout/_macro/post.swig

```diff
@@ -79,14 +79,16 @@
             {% endif %}
 
             {% set date_updated_diff = date(post.date, config_date_format) != date(post.updated, config.date_format) %}
-            {% if theme.post_meta.created_at and theme.post_meta.updated_at %}
+            {% if theme.post_meta.updated_at %}
               {% if !theme.post_meta.updated_diff || theme.post_meta.updated_diff &&  date_updated_diff %}
                 {% set display_updated = true %}
               {% endif %}
             {% endif %}
 
             {% if display_updated %}
```

Comment:
Good jobs, I'd like this feature. 👍 

But I think here `{% if display_updated %}` should be `{% if display_updated || !theme.post_meta.created_at %}`, otherwise any datetime would not be shown when the article's **created time equals its updated time** and the `created_at == false` and `updated_at == true` and `updated_diff == true`

PS: this config is not normalize but I think it could be a situation. (on the test-case? 😃)

Reply:
> theme.post_meta

```yml
# Post meta display settings
post_meta:
  item_text: true
  created_at: true
  updated_at: false
  categories: true
```

This is default configuration.